### ISSUE TITLE
mini player position with zero navbar tabs

### ISSUE THREAD

**Issue by NeeRaj-2401** [libre-tube/LibreTube#2548]
### Steps to reproduce

1. Go to settings > appearance > Navbar order
2. uncheck all the navbar option
3. play any video
4.  swipe down to switch to miniplayer

### Expected behavior

video mini player should be at the bottom part of screen as there is no navbar

### Actual behavior

due to absence of navbar the mini player floats in between

<img src="https://user-images.githubusercontent.com/88180887/210131064-2c935ac0-30f5-4f80-bdd8-d7d78aa618a1.png" width="40%">

### LibreTube version

0.10.0 & latest debug 

### Android version

Android 12

### Other details

disabled navbar because we can access almost eveything from the homescreen itself

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by NeeRaj-2401** [comment]
Hey Bnyro, after #3866 PR and #3766 discussion. Can u please work on this issue. 

i.e. to manage mini player in case of zero navbar tabs
